docs/Customization/Memory-Hierarchy.rst

```diff
@@ -16,6 +16,12 @@ configure 4 KiB direct-mapped caches for L1I and L1D.
 
     import freechips.rocketchip.subsystem.{WithNMediumCores, WithNSmallCores}
```

Comment:
I would also potentially add a "Note: These configurations remove the L2" or something similar in this section.